tests/flytekit/unit/clients/test_raw.py

```diff
@@ -31,5 +31,7 @@ def test_refresh_credentials_basic(mock_credentials_access, mock_requests):
     os.environ[_CREDENTIALS_SECRET.env_var] = "asdf12345"
 
     mock_client = mock.MagicMock()
+    mock_client.url.return_value = 'flyte.corp.net'
```

Comment:
I know that looks cool but I've heard horror stories before about URLs used in tests getting abused by someone buying the domain...

```suggestion
    mock_client.url.return_value = 'flyte.localhost'
```